ipfsapi/http.py

```diff
@@ -38,6 +38,56 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def stream_decode_iter(response, parser):
+    try:
+        for data in response.iter_content(chunk_size=None):
+            #PY2: Use `yield from` instead
```

Comment:
So you're saying that "if we didn't have to support python2, we'd use that syntax"?